cmake/Modules/OffloadOpenmpCommon.cmake

```diff
@@ -0,0 +1,51 @@
+# Check if OMPT support is available
+# Currently, __builtin_frame_address() is required for OMPT
+# Weak attribute is required for Unices (except Darwin), LIBPSAPI is used for Windows
+if(NOT LIBOMP_ARCH)
```

Comment:
Why do we need LIBOMP_ARCH here?

Reply:
Because right below LIBOMP_HAVE_OMPT_SUPPORT needs it and the standalone build of offload is not aware of LIBOMP_ARCH.

Reply:
Isn't there some header that's only built if we have OMPT support? Presumably the presence of that would indicate that whoever built OpenMP had OMPT support, or we could just make it an option, there's no explicit need for this to be automatically detected since a standalone build is supposed to be manual.

Reply:
At first thought this could work. If the ompt header is found offload standalone would set the OMPT support to ON. This would allow us to drop the majority of this top level cmake module. Then that makes the argument for removing it entirely and just put `LIBOMP_HAVE_VERSION_SCRIPT_FLAG` and `pythonize_bool` directly into offload/CMakeLists.txt.